src/content/topics/home/flags-in-codebase/code-references/index.mdx

```diff
@@ -57,6 +57,18 @@ To set up code references in LaunchDarkly, you must have the following prerequis
 
 You can view existing code references for feature flags in the flag dashboard. To view existing code references, visit a feature flag's **Code references** tab.
 
+You can visit this page either by clicking the tab when viewing a flag page, or directly by clicking the code reference count on the flag dashboard.
+
+<Callout intent="info">
+<CalloutTitle>Code reference count may not be visible</CalloutTitle>
+<CalloutDescription>
+
+ Code reference count shows when there is at least one code reference for that flag, or when [filtering by code reference count](/home/organize/dashboard#filtering-feature-flags)
```

Comment:
The styling of this feature may change soon (see [this story](https://app.shortcut.com/launchdarkly/story/152268/add-tooltip-hover-for-code-references)